Snippets/ChatResourceCenter/Resources/.create-keysset.js

```diff
@@ -1,6 +1,10 @@
 const path = require('path');
 const fs = require('fs');
 
+if (!process.env.CI) {
```

Comment:
This script actually run only on Travis using `.travis.yml`. Is there really need to make this?

Reply:
Good point. I'll remove that.

Reply:
resolved in 000c63bfb145bfa55c42f6d8e9a061684cff6986